modules/ROOT/pages/alerts-config.adoc

```diff
@@ -5,17 +5,61 @@ endif::[]
 
 Depending on your subscription level, you can configure different types of alerts:
 
-* Operational alerts for APIs through API Manager
-* Operational alerts for the runtime through Runtime Manager
-* Alerts for graphs in custom dashboards in Anypoint Monitoring.
+* Basic alerts for servers and Mule apps
++
+Limit: Up to 50 for users who do not have a Titanium subscription to Anypoint Platform. If you have a Titanium subscription, the limit is 100 basic alerts, but the limit increases for each vCore. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
+* Advanced alerts for graphs in custom dashboards in Anypoint Monitoring
 +
 Limit: Up to 20 alerts per organization.
 +
 Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the General tab for the graph. If the value is above the threshold, the alert is triggered. The alert is triggered only if the alert state changes (from OK state to Alerting state or vice versa).
+* Operational alerts for APIs through API Manager
+* Operational alerts for the runtime through Runtime Manager
+
+[[basic-alerts]]
+== Set up Basic Alerts for Servers and Mule Apps
+
+You can set up basic alerts to trigger email notifications when a metric you are measuring reaches a specified threshold.
+
+You can create basic alerts for the following metrics for servers or Mule apps:
+
+* For servers:
+** CPU utilization
+** Memory utilization
+** Thread count
+* For Mule apps:
+** Message count
+** Message error count
+** Message response time
+
+=== Configure a Basic Alert
+
+When you create a basic alert, you can select a source for the basic alert, configure the condition under which the basic alert is triggered, and specify which email addresses receive notifications for basic alert triggers.
+
+To create a basic alert:
+
+. Navigate to Anypoint Monitoring.
+. Click the *Alerts* tab.
+. Click the *+ New Alerts* button.
+. Click *Basic alert*.
+. In the *Add basic alert* window, enter the alert name, alert source, alert condition, and alert recipients for your basic alert.
+. Click *Save*.
+
+image::basic-alerts-config.png[Configure a Basic Alert]
 
-== Setting Up API and Runtime Alerts
+[[advanced-alerts]]
+== Set up Advanced Alerts for Graphs in Custom Dashboards
 
-From Alerts in Anypoint Monitoring, you can link out to configuration pages for API and Runtime alerts.
+*https://www.mulesoft.com/anypoint-pricing[Titanium Subscription Required]*
+
+You can set up advanced alerts to trigger email notifications when a data point you are measuring (`JVM - CPU % Utilization`) exceeds a specified threshold. These advanced alerts are listed in the *Alerts* page in Anypoint Monitoring.
```

Comment:
exceeds a specified threshold
Alerts can be triggered by values above or below a certain threshold.

Reply:
```suggestion
You can set up advanced alerts to trigger email notifications when a data point you are measuring (`JVM - CPU % Utilization`) passes a specified threshold. These advanced alerts are listed in the *Alerts* page in Anypoint Monitoring.
```